### ISSUE TITLE
Add in Cubic slashing rate to Namadillo Validator tables

### ISSUE THREAD

**Issue by ChrisHoltDesign** [anoma/namada-interface#1444]
Next to commission if we can add in the cubic slashing rate for each validator

- This would need to appear in the staking flow
- Appear in the staking page

<img width="172" alt="Screenshot 2024-12-23 at 18 30 18" src="https://github.com/user-attachments/assets/a3dd13d1-d3b0-4bb5-b4c2-71720e8a6a45" />

An addition would also be a warning indicator when the slashing rate is high.

---------
We should also add a rollover tooltip here explaining what CSR is

'Cubic slashing rate (CSR)... @brentstone maybe you can right a little sentence her to the tooltip

**Comment by ChrisHoltDesign** [comment]
Reference https://explorer75.org/namada/validators

**Comment by brentstone** [comment]
It might be better to frame this as the `minimum slashing rate` or similar, as that is truly what it is. Since the cubic slashing rate can increase depending on other infractions committed with the respect to the one in questions

**Comment by mateuszjasiuk** [comment]
@brentstone is this slashing rate the thing that we are getting from?
```rs
                    RPC.vp().pos().slashes(context.client()).await,

pub struct Slash {
    /// Epoch at which the slashable event occurred.
    pub epoch: Epoch,
    /// Block height at which the slashable event occurred.
    pub block_height: u64,
    /// A type of slashable event.
    pub r#type: SlashType,
    /// The cubic slashing rate for this validator
    pub rate: Dec,
}

```

**Comment by brentstone** [comment]
@mateuszjasiuk No, we should maybe create a simple API for this, but it is always the value of:

`max [ min_slash_rate,  9 * (validator voting power / total consensus voting power)^2 ]`

The `min_slash_rate` should be the smallest of the protocol parameters `light_client_attack_min_slash_rate` and `duplicate_vote_min_slash_rate`.

It should also be called the MSR (minimum slashing rate), as that is truly what it is. Depending on the presence of other nearby slashes, cubic slashing can make the slash rate larger than this, but it will never be below the expression above.

**Comment by mateuszjasiuk** [comment]
Thanks! I think this can be done on frontend, we should just add min slash rates to the indexer parameters endpoint.
